common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -245,16 +245,21 @@
                 cmr-example-collection-id (str "C1234567-" site-example-provider)
                 doc-type (nth (re-find #"/(.*)/" (str page)) 1)
                 generic-doc-body (read-generic-markdown doc-type)
-                generic-doc-toc (read-generic-markdown-toc (gdocs/all-generic-docs-toc doc-type options))]
+                generic-doc-toc (-> doc-type
+                                    (gdocs/all-generic-docs-toc options)
+                                    (read-generic-markdown-toc)
```

Comment:
no need to have parenthesis here and the next line.

Reply:
Is that our style requirement? It reminds me to much of ruby where function () are optional. I like my functions to look like functions :)
Having said that, I am making the change now.